proto/iotics/api/twin.proto

```diff
@@ -203,6 +210,12 @@ message DescribeTwinResponse {
 
     // optional - if present indicates this response comes from a remote host
     HostID remoteHostId = 3;
+
+    // Twin createdAt timestamp.
+    google.protobuf.Timestamp createdAt = 4;
```

Comment:
These probably belong inside the MetaResult struct for this response

Reply:
Sure - I've moved them